### PR TITLE
Issue #3349917 by chmez: As a CM I want to create Custom content list blocks without subtitle

### PR DESCRIPTION
**Pull request by ghost**
## Problem
Currently, the subtitle is a required field. We should make it non-required so that CM can leave it empty if it is not necessary.

## Solution
As a user with the rights to create it, I can create a Custom content list with no a subtitle:
- A subtitle is not required.
- In the front end, the subtitle section is hidden when no subtitle is provided and the padding is adjusted.

## Issue tracker
- https://www.drupal.org/project/social/issues/3349917
- https://getopensocial.atlassian.net/browse/PROD-24020

## How to test
- [ ] Enable the Social Content Block module.
- [ ] Log in as SM and navigate to the Structure > Block layout > Add custom block.
- [ ] When you click "Custom content list block" then you should see that the "Subtitle" field is optional.

## Release notes
Make the subtitle field of the custom content list block optional.

### CODE REVIEW

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr3357-jeg2lfzqlk3sgkoezz7t3a1o7rq2oyo8.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/641c51cd3bc1c7cf89213e2b

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr3357-jeg2lfzqlk3sgkoezz7t3a1o7rq2oyo8.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/641c51cd3bc1c7cf89213e2b

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr3357-jeg2lfzqlk3sgkoezz7t3a1o7rq2oyo8.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/641c51cd3bc1c7cf89213e2b

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr3357-jeg2lfzqlk3sgkoezz7t3a1o7rq2oyo8.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/641c51cd3bc1c7cf89213e2b

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr3357-jeg2lfzqlk3sgkoezz7t3a1o7rq2oyo8.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/641c51cd3bc1c7cf89213e2b

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr3357-jeg2lfzqlk3sgkoezz7t3a1o7rq2oyo8.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/641c51cd3bc1c7cf89213e2b

**Review by ribel — APPROVED** [review]
Changes was tested and looks good to me.

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr3357-jeg2lfzqlk3sgkoezz7t3a1o7rq2oyo8.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/641c51cd3bc1c7cf89213e2b

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr3357-jeg2lfzqlk3sgkoezz7t3a1o7rq2oyo8.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/641c51cd3bc1c7cf89213e2b

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr3357-jeg2lfzqlk3sgkoezz7t3a1o7rq2oyo8.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/641c51cd3bc1c7cf89213e2b

**Comment by open-social-tugboat** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr3357-jeg2lfzqlk3sgkoezz7t3a1o7rq2oyo8.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/641c51cd3bc1c7cf89213e2b

File: phpstan-baseline.neon [hunk 1]
```diff
@@ -4665,11 +4665,6 @@ parameters:
 			count: 1
 			path: modules/social_features/social_comment_upload/src/SocialCommentUploadConfigOverride.php
 
-		-
```
Comment (ronaldtebrake) [inline phpstan-baseline.neon:4668]: Love to see this 😍 appreciated!!

**Comment by ronaldtebrake** [comment]
Hi @OH17L \\cc @ribel 

Thanks for the improvements and work! I've tested it on Dashboards & landing pages.

It seems to do what it's supposed to do, however on landing pages there are some theming issues.
Is that something you noticed as well?

Dashboard:

Screenshot 2023-03-31 at 14 03 17

Landing page:

Screenshot 2023-03-31 at 14 03 24

---
[EDIT]

Also testing it with #3338 

Seems like there is still an issue with the background.

Screenshot 2023-03-31 at 14 37 41

I'll create a follow up for this one.

**Pull request merged**